roles/os_hardening/tasks/profile.yml

```diff
@@ -13,3 +13,18 @@
     path: /etc/profile.d/pinerolo_profile.sh
     state: absent
   when: os_security_kernel_enable_core_dump | bool
+
+- name: Add env security to profile
+  template:
+    src: 'etc/profile.d/os-security.j2'
+    dest: '/etc/profile.d/os-security.sh'
+    owner: 'root'
+    group: 'root'
+    mode: '0750'
+  when: not os_profile_security_enabled | bool
```

Comment:
the `when` conditions seem mixed up, or am I missing something. 
When `not enabled == true` should delete the file and when `enabled == true` should create the files.

Reply:
correct, I tested it again with a virtual machine. I fixed this with a new commit.